go/adbc/driver/snowflake/driver_test.go

```diff
@@ -672,3 +676,88 @@ func (suite *SnowflakeTests) TestUseHighPrecision() {
 	suite.Equal(1234567.89, rec.Column(1).(*array.Float64).Value(0))
 	suite.Equal(9876543210.99, rec.Column(1).(*array.Float64).Value(1))
 }
+
+func TestUnencryptedJwtAuthenticationUsingString(t *testing.T) {
+	drv := gosnowflake.SnowflakeDriver{}
+
+	uri, ok := os.LookupEnv("SNOWFLAKE_URI")
+
+	if !ok {
+		panic("Cannot find the `SNOWFLAKE_URI` value")
+	}
```

Comment:
Because the `SNOWFLAKE_URI` secret can't be set in forks and other repos, can we just do `t.Skip()` instead of panic'ing?